src/js/outreach/entities-service/entities/outreach.js

```diff
@@ -0,0 +1,63 @@
+import Store from 'backbone.store';
+import BaseCollection from 'js/base/collection';
+import BaseModel from 'js/base/model';
+import fetcher, { handleJSON } from 'js/base/fetch';
+
+const TYPE = 'outreach';
+
+const _Model = BaseModel.extend({
+  type: TYPE,
+  urlRoot: '/api/outreach',
+  registerPatient(attributes) {
+    return fetcher('/api/outreach', {
+      method: 'POST',
+      data: JSON.stringify({
+        data: {
+          type: 'patients',
+          attributes,
+        },
+      }),
+    })
+      .then(handleJSON);
+  },
+  requestOtpCreation() {
+    return fetcher('/api/outreach/otp', {
+      method: 'POST',
+      data: JSON.stringify({
+        data: {
+          type: 'patient-actions',
+          id: this.id,
+        },
+      }),
+    })
+      .then(handleJSON);
+  },
+  requestOtpAuth(code) {
+    const url = '/api/outreach/auth';
```

Comment:
this should be `return this.save({ code }, { url });`. This is an outreach model so we should use the model api